app/Http/Controllers/Web/Admin/Tools/SearchToolsController.php

```
@@ -413,16 +413,13 @@ public function findProjectsStorageUsed($threshold)
         );
 
         //get handle of empty file just created
-        $CSVfilepathOver = Storage::disk('debug')
-                ->path('')
+        $CSVfilepathOver = config('filesystems.disks.debug.root') . '/'
```

PHPMD (2.15.0)

The variable $CSVfilepathOver is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)